pkg/kubelet/metrics/metrics.go

```diff
@@ -35,6 +35,7 @@ const (
 	NodeLabelKey                       = "node"
 	PodWorkerDurationKey               = "pod_worker_duration_seconds"
 	PodStartDurationKey                = "pod_start_duration_seconds"
+	PodStartSLODurationKey             = "pod_start_slo_duration_seconds"
```

Comment:
Technically this is an SLI, not an SLO. An SLO involves thresholds of some sort (i.e. an objective) while the measurement itself is considered an indicator.